packages/client/src/index.tsx

```diff
@@ -19,6 +20,11 @@ if (!API_URL) {
 const httpService = new APIService(API_URL);
 const historyService = new HistoryService();
 
+// init Google Analytics
+if (process.env.NODE_ENV === 'production') {
+  ReactGA.initialize(process.env.GA_TRACKING_CODE || '');
```

Comment:
It would be safe if we don't initialize at all if GA_TRACKING_CODE is undefined as some people may still want to fork the repo and use it without any tracking if they wish